ddcommon/src/container_id.rs

```diff
@@ -113,6 +178,33 @@ pub fn get_container_id() -> Option<&'static str> {
     CONTAINER_ID.as_deref()
 }
 
+/// Returns the `cgroup_inode` if available, otherwise `None`
+pub fn get_cgroup_inode() -> Option<&'static str> {
```

Comment:
This one doesn't need to be public. 
Also, I don't know if in Rust we tend to put public methods at the top and private below. But if you have followed libdd format, that's fine by me.

Reply:
You're right I fixed that. I don't know about any convention in Rust so I put them next to the existing ones.